Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/overlay/InputOverlay.java

```diff
@@ -569,14 +621,15 @@ private void saveControlPosition(int sharedPrefsId, int x, int y)
 	 * InputOverlayDrawableButton. Simply add it to the HashSet of overlay items and wait
 	 * for Android to call the onDraw method.
 	 * 
-	 * @param context  The current {@link Context}.
-	 * @param resId    The resource ID of the {@link Drawable} to get the {@link Bitmap} of.
-	 * @param buttonId Identifier for determining what type of button the initialized InputOverlayDrawableButton represents.
+	 * @param context        The current {@link Context}.
+	 * @param defaultResId   The resource ID of the {@link Drawable} to get the {@link Bitmap} of (Default State).
+	 * @param pressedResId   The resource ID of the {@link Drawable} to get the {@link Bitmap} of (Pressed State).
```

Comment:
OCD is https://en.wikipedia.org/wiki/Obsessive%E2%80%93compulsive_disorder (but people often (incorrectly) say "OCD" even they aren't talking about the actual disorder, but rather are talking about being picky about details – I'm not sure which of the meanings Seanny used)

We do not have a rule about the number of spaces between the param name and description.